spec/controllers/teams_controller_spec.rb

```diff
@@ -64,6 +64,56 @@
         end
       end
 
+      describe "#manage_users" do
+        let(:user1) { create :user, :with_team_and_is_admin }
+        let(:user2) { create :user, :with_team_and_is_admin }
+        it "list users this team to manage" do
```

Comment:
`it "only list users with a team to manage" do`
To better test this you would need to also create users without a team